### PR TITLE
Rename line to port

### PR DESCRIPTION
**Pull request by hujw77**
- [x] Rename line to port
- [x] ORMPPort on testnet
- [x] Support Pangolin

### CODE REVIEW

**Comment by hackfisher** [comment]
@hujw77 The CI failed, would you please give a check?

File: script/deploy/DeployORMPPort.s.sol [hunk 1]
```diff
@@ -56,18 +56,18 @@ contract DeployORMPLine is Common {
         // setConfig();
 
         ScriptTools.exportContract(outputName, "DAO", dao);
-        ScriptTools.exportContract(outputName, "ORMP_LINE", ADDR);
+        ScriptTools.exportContract(outputName, "ORMP_port", ADDR);
```
Comment (hackfisher) [inline script/deploy/DeployORMPPort.s.sol:59]: why lower case here?

File: script/deploy/DeployPortRegistry.s.sol [hunk 2]
```diff
@@ -30,18 +30,18 @@ contract DeployLineRegistry is Common {
...
 
-        instanceId = vm.envOr("INSTANCE_ID", string("deploy_line_registry.c"));
-        outputName = "deploy_line_registry.a";
+        instanceId = vm.envOr("INSTANCE_ID", string("deploy_port_registry.c"));
+        outputName = "deploy_port_registry.a";
         config = ScriptTools.readInput(instanceId);
         c3 = ScriptTools.readInput("../c3");
-        ADDR = c3.readAddress(".LINEREGISTRY_ADDR");
-        SALT = c3.readBytes32(".LINEREGISTRY_SALT");
+        ADDR = c3.readAddress(".portREGISTRY_ADDR");
```
Comment (jiguantong) [inline script/deploy/DeployPortRegistry.s.sol:43]: Upper case?

File: script/deploy/DeployPortRegistry.s.sol [hunk 3]
```diff
@@ -30,18 +30,18 @@ contract DeployLineRegistry is Common {
...
+        SALT = c3.readBytes32(".portREGISTRY_SALT");
```
Comment (jiguantong) [inline script/deploy/DeployPortRegistry.s.sol:44]: Upper case?

File: src/PortRegistry.sol [hunk 4]
```diff
@@ -21,46 +21,46 @@ import "@openzeppelin/contracts/access/Ownable2Step.sol";
...
 import "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
 
-/// @title LineRegistry
-/// @notice LineRegistry will be deployed on each chain.
-/// - Could be used to verify whether the line has been registered.
-/// - Lines that be audited by MsgDAO is marked as `trusted`.
-contract LineRegistry is Initializable, Ownable2Step, UUPSUpgradeable {
-    event SetLine(uint256 chainId, bytes4 code, address line);
-    event DeleteLine(uint256 chainId, bytes4 code, address line);
+/// @title PotRegistry
```
Comment (jiguantong) [inline src/PortRegistry.sol:24]: typo

File: src/examples/ExampleReceiverDapp.sol [hunk 5]
```diff
@@ -20,7 +20,7 @@ pragma solidity ^0.8.17;
 import "../user/Application.sol";
 
 contract ExampleReceiverDapp is Application {
-    event DappMessageRecv(uint256 fromChainId, address fromDapp, address localLine, bytes message);
+    event DappMessageRecv(uint256 fromChainId, address fromDapp, address localPort, bytes message);
 
     // local line address
     address public immutable LINE;
```
Comment (jiguantong) [inline src/examples/ExampleReceiverDapp.sol:26]: LINE => PORT

File: src/examples/ExampleReceiverDapp.sol [hunk 6]
```diff
@@ -32,13 +32,13 @@ contract ExampleReceiverDapp is Application {
...
 
-    /// @notice You could check the fromDapp address or messageLine address.
+    /// @notice You could check the fromDapp address or messagePort address.
     function xxx(bytes calldata message) external {
         uint256 fromChainId = _fromChainId();
         address fromDapp = _xmsgSender();
-        address localLine = _msgLine();
-        require(localLine == LINE);
+        address localPort = _msgPort();
+        require(localPort == LINE);
```
Comment (jiguantong) [inline src/examples/ExampleReceiverDapp.sol:40]: LINE => PORT

**Review by jiguantong — CHANGES_REQUESTED** [review]

**Comment by jiguantong** [comment]
Missed to be updated    exclude(lib/**, out/**, .git/**, cache/**, output/**)
- [ ] *.md README & SUPPORTED
- [ ] *.sh add & config & dao
- [ ] *.json  ormp.json
- [ ] *.sol  xxChainIdMapping & ExampleReceiverDapp & GNSPSBytesLib

**Comment by jiguantong** [comment]
https://github.com/darwinia-network/darwinia-msgport/blob/echo/r/script/input/c3.json
https://github.com/darwinia-network/darwinia-msgport/blob/echo/r/bin/add.sh

**Comment by hujw77** [comment]
> @hujw77 The CI failed, would you please give a check?

CI use mainnet Msgport addresses, would be fixed after mainnet deployed.

**Review by jiguantong — APPROVED** [review]

**Review by hackfisher — APPROVED** [review]

**Pull request merged**